packages/eslint-plugin-office-addins/src/rules/load-sync-read.ts

```diff
@@ -0,0 +1,81 @@
+import { TSESTree } from "@typescript-eslint/experimental-utils";
+import { Variable } from "@typescript-eslint/experimental-utils/dist/ts-eslint-scope";
+import { findReferences, OfficeApiReference } from "../utils";
+
+export = {
+  name: "load-sync-read",
+  meta: {
+    type: <"problem" | "suggestion" | "layout">"suggestion",
+    messages: {
+      loadSyncRead:
+        "Call load on '{{name}}' for '{{loadValue}}' followed by context.sync() before reading the object or its properties",
```

Comment:
I would phrase it "Call context.sync() after calling load on '{{name}}' property on the '{{loadValue}}' and before reading the property"

Reply:
Rephased, just used "properties" instead of "property"